substrate/frame/support/src/traits/hooks.rs

```diff
@@ -318,7 +318,6 @@ pub trait IntegrityTest {
 	fn integrity_test() {}
 }
 
-#[cfg_attr(doc, aquamarine::aquamarine)]
```

Comment:
Nice, can we also remove `simple_mermaid` as well, if it is still used anywhere?

Reply:
We're still using `simple_mermaid` in a few places for embedding `.mmd` files, so I think need to keep it